Spanner/tests/System/QueryTest.php

```diff
@@ -170,56 +150,65 @@ public function testBindInt64ParameterWithInt64Class()
     }
 
     /**
-     * covers 26
+     * covers 27
      */
-    public function testBindNullIntParameter()
+    public function testBindFloat64Parameter()
     {
         $db = self::$database;
 
+        $pi = 3.1415;
         $res = $db->execute('SELECT @param as foo', [
             'parameters' => [
-                'param' => null
-            ],
-            'types' => [
-                'param' => Database::TYPE_INT64
+                'param' => $pi
             ]
         ]);
 
         $row = $res->rows()->current();
-        $this->assertNull($row['foo']);
+        $this->assertEquals($pi, $row['foo']);
     }
 
     /**
      * covers 27
      */
-    public function testBindFloat64Parameter()
+    public function testBindFloat32Parameter()
     {
+        $this->skipEmulatorTests();
         $db = self::$database;
 
         $pi = 3.1415;
-        $res = $db->execute('SELECT @param as foo', [
+        $res = $db->execute('SELECT cast(@param as FLOAT32) as foo', [
```

Comment:
do we need the `cast`?

Reply:
We need the cast here as FLOAT32 doesn't have an explicit literal representation. A number format like `dd.ddd` is by default read as FLOAT64, so CASTing to FLOAT32 will tell the query parser that the number should be read as FLOAT32

Reply:
Understood, Thank you for the clarification.